feat(browser): Session replay GA (#16499)
newrelic/docs-website: src/content/docs/browser/new-relic-browser/performance-quality/browser-monitoring-performance-impact.mdx

      </td>
    </tr>

    <tr>
      <td>
        Session replay
      </td>

      <td>
        no
      </td>

      <td>
        yes
      </td>

      <td>
        yes
      </td>

      <td>

>         Session replays are harvested when one of the following occurs:

Reviewer: Why not session replay events? "Session replays" feels odd to me

>         Session replays are harvested when one of the following occurs:

Author: ```suggestion
        Session replay events are harvested when one of the following occurs:
```

        * When the compressed data collected for the session reaches a maximum size of 64 KB.
        * If the user remains on the same page for more than 60 seconds.

>         * The visibility of the page changes (tab change, focus, navigation, etc)

Reviewer: The first two bullets complete the sentence started in the pre-colon clause, but the last bullet doesn't.

How about "When the visibility..." instead?

>         * The visibility of the page changes (tab change, focus, navigation, etc)

Author: ```suggestion
        * When visibility of the page changes (tab change, focus, navigation, etc)
```

>         Session replay events are harvested when one of the following occurs:
>         * When the compressed data collected for the session reaches a maximum size of 64 KB.
>         * If the user remains on the same page for more than 60 seconds.
>         * When visibility of the page changes (tab change, focus, navigation, etc)

Author: ```suggestion
        Session replay events are harvested when one of the following occurs:
        * When the compressed data collected for the session reaches a maximum size of 64 KB.
        * If the user remains on the same page for more than 60 seconds.
        * When visibility of the page changes (tab change, focus, navigation, etc)
        
The endpoint can differ depending on the locale, but an example for US would be `https://bam.nr-data.net/browser/blobs`.
```
